lock-impl/src/main/java/com/palantir/lock/impl/LegacyTimelockService.java

```diff
@@ -193,7 +191,7 @@ private LockRefreshToken lockAnonymous(LockRequest lockRequest) {
         return locks;
     }
 
-    public static class LockRefreshTokenV2Adapter implements LockTokenV2 {
+    public static class LockRefreshTokenV2Adapter implements LockToken {
```

Comment:
Don't know if we want to rename this, though I'm really not sure what a better name would be...